### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by banlon-jones**
### Home page

- When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
- Along with the category name, you will display some numeric values per category that come from the API.
When the user clicks (or taps) on a category item, the application navigates to the details page.
### Details page

- On the details page, the web app retrieves data from the API to show detailed data in the category.
- When the user clicks on the "Back" button (<), the user navigates to the home page.
### Testing requirements

- Create unit tests for pure functions in your Redux code.
- Create integration tests for your application using the React Testing Library.

https://www.loom.com/share/d1e224bc30624e3cb4c9016987f59f21

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <StatByCountryForm />
+      </section>
+      <section>
+        <Regions regions={regions} />
+      </section>
+    </div>
+  );
+};
+
+export default Home;
```
Comment (omar-labana) [inline src/pages/Home.js]: - [x] As you mention in the pr description, users should be able to filter through states by state name for example. Kindly add the filter component and I'd like to suggest using a controlled react component for this feature.

**Review by omar-labana — COMMENTED** [review]
Hi @banlon-jones, 

Excellent work so far! 👌
You have implemented almost all the required features correctly and without any errors or bugs. ✔️ 

There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Home.js (hunk 1 above)
Comment (banlon-jones) [reply src/pages/Home.js]: Thanks for the feedback I have implemented the filter by name feature, 
thanks for pointing it out :+1:

**Review by elyor-doniyorov — APPROVED** [review]
Hi @banlon-jones, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**